wails-frontend/src/utils/massaFormat.tsx

```diff
@@ -1,5 +1,7 @@
 import { fromMAS, toMAS } from '@massalabs/massa-web3';
 
+const MASSA_SCALING_FACTOR = 9;
```

Comment:
Is that decimals!? if yes, why not:
```suggestion
const MASSA_DECIMALS = 9;
```